client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -176,6 +176,95 @@ export const PatternLibrary = ( {
 
 	const isHomePage = ! category && ! searchTerm;
 
+	const PATTERN_SEO_CONTENT: Record< Category[ 'name' ], { title: string } > = {
```

Comment:
Me removing the `useMemo` call here might have been somewhat subjective, but I don't think these `translate` calls are expensive enough to make us gain much from `useMemo`. This article is something I regularly refer to when `useCallback` and `useMemo` comes up: https://kentcdodds.com/blog/usememo-and-usecallback

Reply:
Thanks @fredrikekelund! That's an article I find myself circling back to pretty regularly as well. I think I just mirrored `useMemo` from the `Themes` [implementation](https://github.com/Automattic/wp-calypso/blob/975b3c2a995d3f39bd8e1904d1b90c0276133116/client/my-sites/themes/use-theme-showcase-logged-out-seo-content.js#L29-L35) I was imitating and could have evaluated the need for it a bit more, but I agree with your point that it probably isn't needed here.